src/assets/scripts/client/constants/aircraftConstants.js

```diff
@@ -253,3 +242,40 @@ export const PERFORMANCE = {
      */
     TYPICAL_CLIMB_FACTOR: 0.7
 };
+
+/**
+ * Separation distances between aircraft
+ *
+ * @property SEPARATION
+ * @type {Object}
+ * @final
+ */
+export const SEPARATION = {
+    /**
+     * 14.816km = 8nm (max possible sep minmum)
+     *
+     * @memberof SEPARATION
+     * @property MAX_KM
+     * @type {number}
+     * @final
+     */
+    MAX_KM: 14.816,
+    /**
+     * Standard Basic Lateral Separation Minimum
+     *
+     * @memberof SEPARATION
+     * @property MIN_STANDARD_LATERAL_KM
+     * @type {number}
+     * @final
+     */
+    MIN_STANDARD_LATERAL_KM: 5.556, // 3nm
```

Comment:
This isn't really a minimum, so to speak. The minimum acceptable separation in standard cases, yes, but in certain cases the separation can be lower. So calling it minimum I think is slightly misleading. Just `STANDARD_LATERAL_KM` should be good here.